Makefile

```diff
@@ -46,6 +45,5 @@ bench:
 pprof-cpu:
 	go tool pprof ./go-jmespath.test ./cpu.out
 
-test-internal-testify:
-	cd internal/testify && go test ./...
-
+install-golint:
+	go install golang.org/x/lint/golint@latest
```

Comment:
Technically this modifies your go.mod file in the project to add this dependency.

`go run` can be used to invoke commands without actually having to install these explicitly.

So for example you can just do `go run golang.org/x/lint/golint <arguments>` wherever you need to run go linting. You can see in our V2 SDK repo we use this pattern to utilize some of our repository release tooling without actually having it modify our dependency descriptor: https://github.com/aws/aws-sdk-go-v2/blob/main/Makefile#L424

Reply:
Np I'll update, just want to make sure I understand the point about go.mod (trying to catch up with this module stuff), are you talking about the go.mod in this repo? When I run `go install ...` I see that it adds the binary to the global `~/go/bin/` dir, but it's not modifying any files in this repo.

```
/tmp/testitout/go-jmespath (sync-tests u=) $ git status
On branch sync-tests
Your branch is up to date with 'origin/sync-tests'.

nothing to commit, working tree clean
/tmp/testitout/go-jmespath (sync-tests u=) $ golint
bash: golint: command not found
/tmp/testitout/go-jmespath (sync-tests u=) $ go install golang.org/x/lint/golint@latest
/tmp/testitout/go-jmespath (sync-tests u=) $ golint -h
Usage of golint:
	golint [flags] # runs on package in current directory
	
      ...

/tmp/testitout/go-jmespath (sync-tests u=) $ which golint
/Users/j/go/bin/golint
/tmp/testitout/go-jmespath (sync-tests u=) $ git status
On branch sync-tests
Your branch is up to date with 'origin/sync-tests'.

nothing to commit, working tree clean
/tmp/testitout/go-jmespath (sync-tests u=) $ cat go.mod
module github.com/jmespath/go-jmespath

go 1.14

require github.com/jmespath/go-jmespath/internal/testify v1.5.1
```

Reply:
Oh never mind, I was incorrect, I thought it did for some reason. You are good then 👍